src/Component/Pages/PostPage/PaginatedPosts.tsx

```diff
@@ -0,0 +1,66 @@
+import { useEffect, useState } from "react";
+import Post from "./Post";
+import PaginationRounded from "./PaginationRounded";
+
+interface IPost {
+  userId: number;
+  id: number;
+  title: string;
+  body: string;
+}
+
+const PaginatedPosts: React.FC = () => {
+  const [posts, setPosts] = useState<IPost[]>([]);
+  const [currentPageNo, setCurrentPageNo] = useState(1);
+  const postsPerPage = 10;
+  const [loadingData, setLoadingData] = useState(false);
+  const [error, setError] = useState<null | string>(null);
+
+  useEffect(() => {
+    setLoadingData(true);
+    fetch("https://jsonplaceholder.typicode.com/posts")
```

Comment:
Use paginated things of jsonplaceholder: 
https://jsonplaceholder.typicode.com/posts?_start=10&_limit=10